src/registrar/templates/admin/transfer_user.html

```diff
@@ -0,0 +1,260 @@
+{% extends 'admin/base_site.html' %}
+{% load i18n static %}
+
+{% block content_title %}<h1>Transfer user</h1>{% endblock %}
+
+{% block extrastyle %}
+
+{{ block.super }}
+
+{% endblock %}  
+
+{% block extrahead %}
+    {{ block.super }}
+    <!-- Making the user select a combobox: -->
+    <!-- Load Django Admin's base JavaScript. This is NEEDED because select2 relies on it. -->                         
+    <script src="{% static 'admin/js/vendor/jquery/jquery.min.js' %}"></script>
+    
+    <!-- Include Select2 JavaScript. Since this view technically falls outside of admin, this is needed. -->
+    <script src="https://cdn.jsdelivr.net/npm/select2@4.1.0-rc.0/dist/js/select2.min.js"></script>
```

Comment:
This is what I was referencing earlier. If hypothetically the developers of select2 got hacked (or were deliberately malicious) they could upload arbitrary code at this url and it would execute in the users browser. This is a risk we take with basically all our dependencies, so it's not unacceptable, but again want to make sure it's carefully considered.

Reply:
select2 makes the django admin combobox possible on the transfer page. It's django source code basically. Without it, the transfer page would have to be registered as a django admin view (which it is not), or we'd need to use USWDS's combobox in this view's context. There might be a way to load this piece of Django's source code from our existing build, but I took a shortcut and used the cdn

Reply:
One option: We download the select2 script and load it from our static folder. Is it worth it though?

Reply:
^ Legit question, you or Alysia would be the ones best qualified to answer.

Reply:
I'm cool with this as-is. Just wanted to make sure there wasn't an easy workaround.